devs/docs/tests.rst

```diff
@@ -10,6 +10,24 @@ Run the all of `doctests`_::
 
     $ ./gradlew itest
 
+For running itest on Windows WSL needs to be installed. If downloaded Linux distro comes without Java, install it by running the following commands::
+
+    $ sudo add-apt-repository ppa:openjdk-r/ppa
+    $ sudo apt-get update
+    $ sudo apt install openjdk-11-jdk.
+
+Then install python virtual environment by running::
+
+    $ sudo apt-get install python3-venv
+
+To support symlinks, enable Developer Mode and run::
+
+    $ git config --global core.symlinks true
+
+After all configuration is done, launch WSL from the project directory and run::
```

Comment:
Can you elaborate how to exactly launch WSL from the project directory?

Reply:
Done